guides/limits.md

```diff
@@ -0,0 +1,26 @@
+# Limits
+
+This guide describes limits of XDN platform as applied to all projects running on it.
+
+## Legend
+
+* `Kb` stands for kilobytes and means 1,024 bytes
+* `Mb` stands for megabytes and means 1,024 kilobytes
+
+## Request and response limits
+
+| Value | Limit | Description |
+| -- | -- | -- |
+| Path and query string size | 8Kb | The maximum bytes (not characters) that XDN will accept in path and query string. |
```

Comment:
what happens if you exceed this? Error? truncation? unknown?

Reply:
I didn't document that as we cannot handle it because Fastly handles it before us:  https://docs.fastly.com/en/guides/resource-limits#request-and-response-limits. I could just say what they say:

> Exceeding the limit results in a Too long request string error.